### PR TITLE
Added Cost Reference Section

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR adds information about references product costs.

### CODE REVIEW

**Comment by bergerhoffer** [comment]
Addresses issue #186.

@redhat-documentation/ccs-style-council for review please!

File: supplementary_style_guide/style_guidelines/general-formatting.adoc [hunk 1]
```diff
@@ -329,3 +329,12 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
...
 image:images/yes.png[yes] It is anticipated that an upcoming release will include a fix for this issue.
+
+[[references-to-cost]]
+== References to cost
+
+Avoid all references to the costs of Red Hat products. These references can confuse users and cause legal concerns. This information is best left to marketing materials.
+
+.Example phrases
+
+* "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to me the product is free when it isn't.
```
Comment (sbmetz) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:340]: Would it be helpful to add an example about avoiding "free" as well as "cost"? Optionally link to IBM SG section on "free" as in original discussion.

Comment (sbmetz) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:340]:
```suggestion
* "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to mean the product is free when it isn't.
```

Comment (dfitzmau) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:340]: Do we need to scope cost a bit more? Do we need to state monetary costs? Learning a product might cost time and resources.

Comment (dfitzmau) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:340]: What about "no cost" or "without charge". These two phrases are mentioned in the IBM SG, but should also be misleading?

Should we state that an IBM SG entry exists for "free", but the RH SSG supersedes the guidance on the use of this term and related terms?

File: supplementary_style_guide/style_guidelines/general-formatting.adoc [hunk 2]
```diff
@@ -329,3 +329,12 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
...
 image:images/yes.png[yes] It is anticipated that an upcoming release will include a fix for this issue.
+
+[[references-to-cost]]
+== References to cost
+
+Avoid all references to the costs of Red Hat products. These references can confuse users and cause legal concerns. This information is best left to marketing materials.
+
+.Example phrases
+
+* "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to mean the product is free when it isn't.
```
Comment (bburt-rh) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:340]:
```suggestion
* "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to mean that the product is free when it is not.
```

**Review by EricPonvelle — COMMENTED** [review]
@bburt-rh - Apologies, but I seemed to have forgotten to actually submit my suggestion to cover "free" as well. Can you review this suggestion?

**Comment by bburt-rh** [comment]
> @bburt-rh - Apologies, but I seemed to have forgotten to actually submit my suggestion to cover "free" as well. Can you review this suggestion?
Your new suggested wording sounds good to me @EricPonvelle. I think pointing to the potential for confusion on a general level works better than the other wording.

File: supplementary_style_guide/style_guidelines/general-formatting.adoc (hunk 1 above)
Comment (dfitzmau) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc]: Sounds good, @EricPonvelle . Thanks!

File: supplementary_style_guide/style_guidelines/general-formatting.adoc [hunk 3]
```diff
@@ -329,3 +329,12 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
 image:images/no.png[no] We will fix this issue in the 18.3.4 release next February.
 
 image:images/yes.png[yes] It is anticipated that an upcoming release will include a fix for this issue.
+
+[[references-to-cost]]
+== References to cost
+
+Avoid all references to the costs and charges of Red Hat products. While the IBM style guide blocks the usage of `free`, these references should also be avoided since they can confuse users and cause legal concerns. Any cost information is best left to marketing materials.
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:336]: Just a few suggestions/clarifications

```suggestion
Avoid all references to the costs and charges of Red Hat products. While the IBM style guide recommends against using the term "free", any references to cost should also be avoided in product documentation since they can confuse users and cause legal concerns. Any cost information is best left to marketing materials.
```

File: supplementary_style_guide/style_guidelines/general-formatting.adoc [hunk 4]
```diff
@@ -329,3 +329,12 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
...
+
+.Example phrases
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:338]:
```suggestion
.Example problematic phrase
```

File: supplementary_style_guide/style_guidelines/general-formatting.adoc [hunk 5]
```diff
@@ -329,3 +329,12 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
...
+
+* "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to mean that the product is free when it is not.
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:340]: I know what you're talking about here - that it was removed during peer review. But other readers might not. Suggest rewording a bit (and adding another comma)

```suggestion
* "at no initial cost" - This phrase should be avoided in documentation because, while it implies there are further costs, it can also be construed to mean that the product is free when it is not.
```

**Review by bergerhoffer — COMMENTED** [review]
@EricPonvelle Thanks a lot for submitting this!

I had a few comments/suggestions if you can please take a look!

**Review by bergerhoffer — DISMISSED** [review]
Thanks for the updates, LGTM now!

File: supplementary_style_guide/style_guidelines/general-formatting.adoc [hunk 6]
```diff
@@ -329,3 +329,12 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
 image:images/no.png[no] We will fix this issue in the 18.3.4 release next February.
 
 image:images/yes.png[yes] It is anticipated that an upcoming release will include a fix for this issue.
+
+[[references-to-cost]]
+== References to cost
+
+Avoid all references to the costs and charges of Red Hat products. While the IBM style guide recommends against using the term "free", any references to cost should also be avoided in product documentation since they can confuse users and cause legal concerns. Any cost information is best left to marketing materials.
```
Comment (ndeevy) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:336]: The general entry looks really good but I'm adding these small suggested edits to make it more aligned with our IBM and writing style for these items:
- Not to use while to mean although
- Not to use since to mean because
- Avoid 'should',  passives, and be prescriptive
- minimalism -for the last sentence. It's debatable but I think it seems a bit like fluff. And in documentation guidelines maybe it's not applicable to discuss what belongs in marketing material?

```suggestion
Avoid all references to the costs and charges of Red Hat products. Although the _IBM Style Guide_ recommends against using the term "free", also avoid any references to cost in product documentation because they can confuse users and cause legal concerns.
```

File: supplementary_style_guide/style_guidelines/general-formatting.adoc [hunk 7]
```diff
@@ -329,3 +329,12 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
...
+
+.Example problematic phrase
+
+* "at no initial cost" - This phrase should be avoided in documentation because, while it implies there are further costs, it can also be construed to mean that the product is free when it is not.
```
Comment (ndeevy) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:340]:
```suggestion
* "at no initial cost" - Avoid this phrase in documentation because, although it implies there are further costs, it can also be construed to mean that the product is free when it is not.
```

suggestion here is for similar reasons listed in the last comment ^^

**Comment by ndeevy** [comment]
Hi @EricPonvelle It looks good but I made a couple of small style suggestions for your consideration.

**Review by dfitzmau — APPROVED** [review]

File: supplementary_style_guide/style_guidelines/general-formatting.adoc [hunk 8]
```diff
@@ -329,3 +329,12 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
 image:images/no.png[no] We will fix this issue in the 18.3.4 release next February.
 
 image:images/yes.png[yes] It is anticipated that an upcoming release will include a fix for this issue.
+
+[[references-to-cost]]
+== References to cost
+
+Avoid all references to the costs and charges of Red Hat products. Although the _IBM Style Guide_ recommends against using the term "free", also avoid any references to cost in product documentation because they can confuse users and cause legal concerns. Any cost information is best left to marketing materials.
```
Comment (Amrita42) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:336]: @EricPonvelle  Suggestion >> Any cost information is best referenced in marketing materials. wdyt?

Comment (EricPonvelle) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc]: @Amrita42 I updated the PR with your suggestion. Thanks!

Comment (Amrita42) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc]: Thanks @EricPonvelle

**Comment by Amrita42** [comment]
lgtm

**Review by Amrita42 — APPROVED** [review]

**Review by ndeevy — APPROVED** [review]

**Comment by ndeevy** [comment]
Thanks @EricPonvelle, this looks ace now :-]

**Review by bergerhoffer — APPROVED** [review]
LGTM too!

**Comment by bergerhoffer** [comment]
3 approvals - merging!

**Comment by jseseCCS** [comment]
hi, @bergerhoffer! please forgive me if i'm doing this wrong. i've been able to pay only partial attention to this glossary process! i wonder if i might suggest an additional term? it was in some content i edited recently. 

bytecode
